Source/Scene/HeadingPitchRange.js

```diff
@@ -18,26 +18,12 @@ define([
      * @param {Number} [heading=0.0] The heading angle in radians.
      * @param {Number} [pitch=0.0] The pitch angle in radians.
      * @param {Number} [range=0.0] The distance from the center in meters.
+     *
+     * @deprecated
      */
     var HeadingPitchRange = function(heading, pitch, range) {
-        /**
-         * Heading is the rotation from the local north direction where a positive angle is increasing eastward.
-         * @type {Number}
-         */
-        this.heading = defaultValue(heading, 0.0);
-
-        /**
-         * Pitch is the rotation from the local xy-plane. Positive pitch angles
-         * are above the plane. Negative pitch angles are below the plane.
-         * @type {Number}
-         */
-        this.pitch = defaultValue(pitch, 0.0);
-
-        /**
-         * Range is the distance from the center of the local frame.
-         * @type {Number}
-         */
-        this.range = defaultValue(range, 0.0);
+        deprecationWarning('HeadingPitchRange', 'Scene/HeadingPitchRange is deprecated. Use Core/HeadingPitchRange instead.');
+        return new CoreHeadingPitchRange(heading, pitch, range);
```

Comment:
@mramato is it legit to `return new` from a constructor function?

Since this code is coding away, plan B can just be to duplicate the `HeadingPitchRange` implementation here since it is small.